.travis.yml

```diff
@@ -30,9 +30,19 @@ script:
   - ./scripts/check-code.sh
 
 
-#deploy:
-# - provider: script
-#   script: scripts/aws-lambda-deploy.js hello-world production
-#   skip_cleanup: true
-#   on:
-#     branch: master
+deploy:
+   provider: script
+   script:
+     # Serverless deploy to production environment after merging to production branch
+     - if [ "$TRAVIS_PULL_REQUEST" == "false" ]
+       then
+          cd lambda_functions/execute_ssh_command_js/
+          npm install -g npm
```

Comment:
Why is this line necessary? How is `npm` supposed to install itself if it's not already installed?

Reply:
Removed this line